runhouse/resources/hardware/cluster.py

```diff
@@ -1124,7 +1124,9 @@ def _start_or_restart_helper(
 
             env_vars = _process_env_vars(default_env.env_vars)
             if env_vars:
-                self.call(default_env.name, "_set_env_vars", env_vars)
+                self.set_process_env_vars(
```

Comment:
does it still make sense to sync config env vars (ex: log level, observability) onto the cluster when syncing the default env, or should those be set as process env vars too?

Reply:
These need to be set in the default env -- better to set before the daemon starts. These env vars are set in the process